plugins/ui/src/js/src/ObjectView.tsx

```diff
@@ -13,10 +13,7 @@ function ObjectView(props: ObjectViewProps) {
   const { object } = props;
   log.info('Object is', object);
 
-  const fetch = useMemo(
-    () => object.fetch.bind(object) as () => Promise<Widget>,
-    [object]
-  );
+  const fetch = useCallback(() => object.fetch() as Promise<Widget>, [object]);
```

Comment:
Don't need the cast at all:
```suggestion
  const fetch = useCallback(() => object.fetch(), [object]);
```

Reply:
You'll need to remove the unused `Widget` import as well.

Reply:
Nope I can't remove the cast.

The `jsapi-types` is `Promise<unknown>`. The `WidgetComponent` expects `Promise<Widget>`. Should probably update the `jsapi-types`, although table and figure don't conform to the `Widget` class, so really it should be `Promise<Widget | Table | Figure>` and we should have some TS type guards. I don't think table and widget have a type on the instances like widgets do.